recipes/fastnumbers/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fastnumbers" %}
+{% set version = "2.0.1" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "ea67263d6360b9294c4bc5924089771d3f003bf6b6343383d700ea9659b506fb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
It's suggested in the example recipe that if a package compiles C code, then `toolchain` should be a build requirement:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33